[2024-11-02T13:33:19.614+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: schema_dag.submit_schemas scheduled__2024-09-16T00:00:00+00:00 [queued]>
[2024-11-02T13:33:19.634+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: schema_dag.submit_schemas scheduled__2024-09-16T00:00:00+00:00 [queued]>
[2024-11-02T13:33:19.634+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2024-11-02T13:33:19.697+0000] {taskinstance.py:1380} INFO - Executing <Task(PinotSchemaSubmitOperator): submit_schemas> on 2024-09-16 00:00:00+00:00
[2024-11-02T13:33:19.712+0000] {standard_task_runner.py:57} INFO - Started process 4139 to run task
[2024-11-02T13:33:19.719+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'schema_dag', 'submit_schemas', 'scheduled__2024-09-16T00:00:00+00:00', '--job-id', '1447', '--raw', '--subdir', 'DAGS_FOLDER/schema_dag.py', '--cfg-path', '/tmp/tmp1c1gw361']
[2024-11-02T13:33:19.725+0000] {standard_task_runner.py:85} INFO - Job 1447: Subtask submit_schemas
[2024-11-02T13:33:19.821+0000] {task_command.py:415} INFO - Running <TaskInstance: schema_dag.submit_schemas scheduled__2024-09-16T00:00:00+00:00 [running]> on host 5e812a6a7c02
[2024-11-02T13:33:20.004+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='minhtranquang' AIRFLOW_CTX_DAG_ID='schema_dag' AIRFLOW_CTX_TASK_ID='submit_schemas' AIRFLOW_CTX_EXECUTION_DATE='2024-09-16T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-09-16T00:00:00+00:00'
[2024-11-02T13:33:20.007+0000] {pinot_schema_operator.py:33} ERROR - An error occured: 'function' object has no attribute 'glob'
[2024-11-02T13:33:20.028+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=schema_dag, task_id=submit_schemas, execution_date=20240916T000000, start_date=20241102T133319, end_date=20241102T133320
[2024-11-02T13:33:20.093+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-11-02T13:33:20.128+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
